covid_xprize/examples/prescriptors/neat/README.md

```diff
@@ -21,7 +21,7 @@ export PYTHONPATH=$PYTHONPATH:<workspace>/covid-xprize/
 Setup pretrained LSTM predictor by copying the sample trained weights file to the
 location that it loads weights from:
 ```
-cd <workspace>/covid-xprize/examples/predictors/lstm/
+cd <workspace>/covid-xprize/covid_xprize/examples/predictors/lstm/
```

Comment:
That doesn't look right. Should a single `covid_xprize`, right?

Reply:
Yup -- either search-and-replace booboo, or I had [semantic satiation](https://en.wikipedia.org/wiki/Semantic_satiation) after seeing `covid_exprize` too many times. 

I'll fix all these once you & Elliot have made your first pass.

Reply:
Oh wait...the first one is with hyphen (the git repo), the second is with underscore (the top level Python package). I think we need both.

Reply:
Yes, you're right